spec/outputs/loggly_spec.rb

```diff
@@ -110,6 +106,46 @@ def logger_for(plugin)
       end
     end
 
+    context 'prepare tags' do
+      it 'failed interpolation' do
```

Comment:
Your `context` and `it` strings should read more like sentences that describe the situation you're setting up (context) and intended result (it).

Here's an example for the two lines above:

```ruby
context 'when computing the tags'
  it 'should not leave a failed interpolation in the tag list' do
```

then of course all following `it` blocks should be clarified to describe expected behaviour of each test case as well.